bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/util/AbstractComponentImpl.java

```diff
@@ -84,6 +85,8 @@ public abstract class AbstractComponentImpl implements Component {
      */
     private String id;
 
+    protected I18n i18n = null;
```

Comment:
pls move initialisation of i18n in this class from AbstractFormComponentImpl.

Reply:
i18n is initialized in the base class, as discussed offline.